src/components/UserRepos/UserRepos.tsx

```diff
@@ -0,0 +1,33 @@
+import { useState } from "react";
+import { Repo } from "../../interface";
+import Button from "../Button/Button";
+import ReposDetails from "../ReposDetails/ReposDetails";
+
+interface repoProps {
+    repos: Repo[]
+}
+
+function UserRepos({ repos }: repoProps) {
+    const [repoDetails, setRepoDetails] = useState<Repo>()
+
+    const showRepoDescription = (repo: Repo) => {
```

Comment:
when you have a handler function, name it towards what it is supposed to do, like:
```suggestion
    const handleRepoClick = (repo: Repo) => {
```